Directory.Build.props

```diff
@@ -20,6 +20,7 @@
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <!-- We use full (Windows PDBs) until cross platform support for source link will get better -->
     <DebugType>full</DebugType>
+     <NoWarn>NU5105</NoWarn>
```

Comment:
Which warning is this suppressing?

Reply:
This is a warning that you are using a nuget version that is not SemVer 1.0.0 compatible.

```
error NU5105: The package version '0.5.0-blazor-0.9.0-preview3-19154-02' uses SemVer 2.0.0 or components of SemVer 1.0.0 that are not supported on legacy clients. Change the package version to a SemVer 1.0.0 string. If the version contains a release label it must start with a letter. This message can be ignored if the package is not intended for older clients.
```